runtime/drivers/registry.go

```diff
@@ -33,4 +35,36 @@ type Instance struct {
 	CreatedOn time.Time `db:"created_on"`
 	// UpdatedOn is when the instance was last updated in the registry
 	UpdatedOn time.Time `db:"updated_on"`
+	// Env contains user-provided environment variables
+	Env *Env `db:"env"`
+	// ProjectEnv contains default environment variables from rill.yaml
+	// (NOTE: This can always be reproduced from rill.yaml, so it's really just a handy cache of the values.)
+	ProjectEnv *Env `db:"project_env"`
```

Comment:
A `map[string]string` is already a pointer, so this can just be `Env`.

Actually, what about not having the `Env` type and just using a `map`? Other code that uses it needs to know it's a `map` anyway (to access values), so the type alias doesn't hide an implementation detail.

Reply:
The idea behind defining a type alias was not to hide impl details but to be able to define `driver.Valuer` and `sql.Scanner` interfaces. 
I could not keep it as `Env` itself since sql driver returned following error(possibly because there is no support for maps):
`Error: sql: converting argument $8 type: unsupported type drivers.Env, a map`

Reply:
Given comment below, I don't think this is needed. But you could have this as `Env` instead of `*Env`, and do `&inst.Env` when passing into SQL.